main/src/main/java/org/mobilitydata/gtfsvalidator/report/model/FeedMetadata.java

```diff
@@ -206,7 +217,24 @@ private void loadPathwayDirectionsComponent(GtfsFeedContainer feedContainer) {
             GtfsPathway.FILENAME,
             List.of(
                 GtfsPathway::hasSignpostedAs,
-                (Function<GtfsPathway, Boolean>) GtfsPathway::hasReversedSignpostedAs)));
+                (Function<GtfsPathway, Boolean>) GtfsPathway::hasReversedSignpostedAs,
+                GtfsPathway::hasMaxSlope,
+                GtfsPathway::hasMinWidth,
+                GtfsPathway::hasLength,
+                GtfsPathway::hasStairCount)));
+  }
+
+  private void loadPathwayExtraComponent(GtfsFeedContainer feedContainer) {
+    specFeatures.put(
+        "Pathway (extra)",
+        hasAtLeastOneRecordForFields(
```

Comment:
I think `hasAtLeastOneRecordForFields` will do an AND of all the specified functions that are passed to it ( hasMaxSlope, hasMinWidth, hasLength and hasStairCount)
But according to this it should be an OR:
```
Add Pathways (extra) = If 1 occurrence of max_slope OR min_width OR length OR stair_count in pathways.txt
```

Reply:
Good catch. The List.of method is a list of conditions that each record must satisfy.